lib/webpacker/helper.rb

```diff
@@ -42,7 +45,9 @@ def javascript_pack_tag(*names, **options)
   #   <%= stylesheet_pack_tag 'calendar', 'data-turbolinks-track': 'reload' %> # =>
   #   <link rel="stylesheet" media="screen" href="/packs/calendar-1016838bab065ae1e122.css" data-turbolinks-track="reload" />
   def stylesheet_pack_tag(*names, **options)
-    stylesheet_link_tag(*sources_from_pack_manifest(names, type: :stylesheet), **options)
```

Comment:
The doc for the method should be updated (or removed? with a reference to the readme.)